src/model/page.urls.ts

```diff
@@ -1,7 +1,14 @@
+import * as template from "./template.paths";
+
+/**
+ * Keep template names in template.paths.ts and reference them in here for URLs
```

Comment:
Think it's worth expanding on the comment to explain why we have both COMPANY_NUMBER and PROMISE_TO_FILE_COMPANY_NUMBER constants (and others). I know we discussed this and understood why but I've forgotten now. Can you remember why @markpit ?

Reply:
PROMISE_TO_FILE_XXX is probably used for redirects

Reply:
comments updated

Reply:
Perhaps say these are used for 'render' calls?